registry/src/skills.json

```
@@ -7,6 +7,92 @@
         "author": null,
         "description": "Enables your agent to fetch token details for any token on Base",
         "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details",
+        "imageUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details/images/logo.png"
```

changing path, this should be the correct one to work properly
```suggestion
        "imageUrl": "https://raw.githubusercontent.com/moxie-protocol/moxie-agent-skills/refs/heads/main/packages/plugin-moxie-token-details/images/logo.png"
```